mindmeld/tokenizer.py

```diff
@@ -170,10 +110,15 @@ def _init_regex(self):
             "apostrophe": (" ", None),
         }
 
-        # Create a regular expression  from the dictionary keys
+        # Create compiled regex expressions
         self.keep_special_compiled = re.compile(
-            "(%s)" % ")|(".join(keep_special_regex_list), re.UNICODE
+            "(%s)"
+            % (
+                ")|(".join(self.config["allowed_patterns"]),
+            ),
```

Comment:
If allowed_patterns doesn't make a well-formed regex here, is the failure interpretable for the developer?